client/components/forms/form-phone-input/test/index.jsx

```diff
@@ -25,52 +24,53 @@ describe( 'FormPhoneInput', () => {
 	const localizeProps = { translate: ( string ) => string };
 
 	describe( 'getValue()', () => {
-		test( 'should set country from props', () => {
-			const phoneComponent = shallow(
+		test( 'should set country from props', async () => {
```

Comment:
Why does it need to be async??

Reply:
It doesn't, good catch 👍🏻
It's a leftover from using `@testing-library/userEvent` v14 which I tried before just using `fireEvent`.

Reply:
Cool!